### DISCUSSION TITLE
Next alpha release?

### DISCUSSION THREAD

**Discussion by TheNando** [alephjs/aleph.js#185] [Q&A] [answered]
It's a been a while since there has been a release. Over a month and 105 commits. What are the target objectives for the next release?

**Comment by shadowtime2000** [comment]
I agree. I think we should release alpha 9 asap because it fixes a couple bugs and makes it compatible with Deno 1.8.0. I believe there were other goals such as fixing up the bundler, #176, but I think we could release alpha 9 as just a couple bug fixes.

cc @ije

**Reply by ije** [reply]
on the way...

**Comment by TheNando** [comment, marked as answer]
New releases are here. Thanks for all your hard work!
